### ISSUE TITLE
[Accessibility]: Code Insights Dashboards - Insight filter and option menus broken

### ISSUE THREAD

**Issue by unclejustin** [sourcegraph/sourcegraph#41689]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33501

### Problem description

![PixelSnap 2022-09-15 at 17 25 47](https://user-images.githubusercontent.com/1855233/190519462-146467b2-956a-4e50-a03d-ab088ba82ccb.png)

Neither of the insight card menus open.

### Expected behavior

Clicking either of the menu buttons should show the drop-down menu.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
